cmd/createcluster.go

```diff
@@ -51,6 +51,7 @@ const (
 	zeroAddress    = "0x0000000000000000000000000000000000000000"
 	defaultNetwork = "mainnet"
 	minNodes       = 3
+	minThreshold   = 2
```

Comment:
Shall this be `ceil(n*2/3)`?

Reply:
I left `minThreshold=2` here for tests and to clearly indicate that is the minimum threshold charon supports (when using 3 nodes only).
But indeed when validating the configuration it checks that the threshold is at least `ceil(n*2/3)`.